public/js/app/components/AnalysisMenu.jsx

```diff
@@ -321,34 +307,42 @@ function AnalysisMenu(props) {
       {/* ************************************************************************************ */}
       {/*                               SEMESTER DROPDOWN                          */}
       {/* ************************************************************************************ */}
-      <Dropdown isOpen={dropdownOpen} toggle={toggleDropdown} className="select-semester">
-        <div className="inline-flex padding-top-30">
+
+      <div className="col-4 nopadding">
+      <div className="inline-flex padding-top-30">
           <h3> {translate.select_semester} </h3>
           <InfoButton addClass="padding-top-30" id="info_select_semester" textObj={translate.info_select_semester} />
+      </div>
+      <form>
+        <div className="form-group">
+          <div className="form-select">
+            <div className="select-wrapper">
+              <select 
+              className="form-control"
+              id="semesterDropdownControl" 
+              aria-label={translate.select_semester} 
+              onChange={handleSelectedSemester}
+              defaultValue={semester && semester > 0 && !firstVisit ? semester : translate.select_semester}
+              key="no-chosen">
```

Comment:
key="no-chosen" should be inside of option tag, not select tag. Because key is used if there are list of components with the same tag.

Reply:
Ah, great thanks! It still a bit confusing to me, how to work with the attributes - will do more reading!